profile_library/shelly/Shelly Wave 1PM Mini/model.json

```
@@ -0,0 +1,21 @@
+{
+  "measure_description": "Manually measured with Gossen Metrawatt 29S (energy speciality DMM)"
```

```suggestion
  "measure_description": "Manually measured with Gossen Metrawatt 29S (energy speciality DMM)",
```

Forgot comma here that causes the JSON to be invalid